atlas/embed.go

```diff
@@ -8,4 +8,7 @@ import (
 var (
 	//go:embed transactions/user/buy_packs_primary_sale.cdc
 	UserBuyPacksPrimarySale []byte
+
+	//go:embed transactions/admin/fulfill_pack_buyback_offer.cdc
+	AdminFulfillPackBuybackOffer []byte
```

Comment:
i just thought of something now, can we add a struct for the tx that satisfies the template variables making it easier for whoever is using the transaction. This way they don't have to look at the file itself

Reply:
we could also expose functions that can be called with parameters and it returns the parsed transaction

Reply:
Good point, exposed functions to make it easier to use